src/main/java/mate/academy/hw01/Rhombus.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.hw01;
+
+public class Rhombus {
+    public static void main(String[] args) {
+        rhombusDisplay();
+    }
+
+    private static void rhombusDisplay() {
+        System.out.println("Size of Rhombus = 3");
```

Comment:
Size should be an argument of method

Reply:
I changed it: https://github.com/MateAcademy/lessons/commit/f04b1602934df1f50ba9d20ce82368a4b18cb43f#diff-9a5065b35bbf915b2c09e81865316b09